package-res/resources/web/pentaho/visual/ccc/abstract/View.js

```diff
@@ -21,22 +21,24 @@ define([
   "cdf/lib/CCC/protovis",
   "../_axes/Axis",
   "../util",
+  "pentaho/util/object",
+  "pentaho/data/filter",
   "pentaho/visual/events",
   "pentaho/visual/color/utils",
   "pentaho/visual/color/paletteRegistry",
   "pentaho/data/TableView",
   "pentaho/i18n!view"
 ], function(View,
-    def, pvc, cdo, pv, Axis,
-    util, visualEvents, visualColorUtils, visualPaletteRegistry,
-    DataView, bundle) {
+            def, pvc, cdo, pv, Axis,
```

Comment:
I'm a formatting maniac. Was there really a need to do the auto formatting? And then no going through all the changes to see if they were ok? Better give up, maybe?